liiatools_pipeline/ops/common_org.py

```diff
@@ -26,9 +27,18 @@ def move_error_report():
 def move_current_and_concat_view(config: CleanConfig):
     current_folder = incoming_folder().opendir("current")
     destination_folder = shared_folder()
+
+    existing_files = destination_folder.listdir("/")
+
+    authority_regex = "|".join(authorities.codes)
+    current_files_regex = f"({authority_regex})_\d{{4}}"
```

Comment:
Are you referring to the {{4}} braces or just in general?

Reply:
Ah yes the {{4}}

Reply:
Ok I thought so, basically when using f"strings if you want to actually output a curly brace you need to use two, so f"{{4}} will result in {4}. I've used that here so the regex reads it as e.g. SOU_\d{4} which means it will look to match exactly 4 digits (in this case they should signify a year e.g. 2021). By this point the files have been renamed by the pipeline so they should follow this pattern exactly

Reply:
Ah yes the f string wrapper - thanks for explaining!